.github/workflows/snyk.yml

```diff
@@ -0,0 +1,23 @@
+name: snyk
+on:
+  schedule:
+    - cron: "0 10 * * 1" # Monday @ 10am UTC
+  workflow_dispatch:
+
+env:
+  SNYK_TOKEN: ${{ secrets.SNYK_TOKEN }}
+  SNYK_ORG: rstudio-connect
+
+jobs:
+  snyk-monitor:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - name: Run Snyk
+        uses: snyk/actions/python@master
+        with:
+          command: monitor
+          args: --file=Pipfile --org=${{ env.SNYK_ORG }}
```

Comment:
Do we need to specify the project name here?

Reply:
You only really need a project name if you have more than one "package" being scanned in a single repo.